lib/xpm_ruby/connection.rb

```diff
@@ -55,6 +55,9 @@ def handle_response(response)
         else
           raise Unauthorized.new(detail)
         end
+      when 403 # this can happen with a bad xero_tenant_id
+        detail = JSON.parse(response.body)["Detail"]
+        raise Unauthorized.new(detail)
```

Comment:
Thoughts on a separate error? For example:

```
class Forbidden < Error; end
```

```
when 403
  raise Forbidden.new(detail)
```

Thinking it might help us troubleshoot in calling code? Api seems to distinguish the two might be worth preserving that given we're conforming.

Reply:
Yeah, that makes sense. :)